content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean.md

```diff
@@ -0,0 +1,120 @@
+---
+title:
+  - 1. Boolean
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Boolean.DT
+  - ib-19.3.Datatype-Primitief-Boolean.DT
+draft: false
+---
+
+> Meer informatie over [[4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean|Boolean]]
+
+## Opdracht 1
+Maak een boolean aan en print het resultaat.
+
+### Specificaties
+- Creëer een `bool` genaamd `IsWaar`.
+- Print de waarde van de `IsWaar uit in de console.
+
+### Verwachte output:
+Als `IsWaar = true`, komt in de console:
+```
+true
+```
+Als `IsWaar = false`, komt in de console:
+```
+false
+```
+
+### Nu jij:
+``` csharp runner
+using System;
+
+Console.WriteLine();
+``` 
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+>
+>bool IsWaar = true;
+>Console.WriteLine(IsWaar);
+> ```
+
+---
+## Opdracht 2
+Draai het resultaat om van een boolean.
+
+### Specificaties
+- Neem de code over van opdracht 1.
+- Draai de gedeclareerde waarde om.
+
+### Verwachte output:
+Als `IsWaar = true`, komt in de console:
+```
+false
+```
+Als `IsWaar = false`, komt in de console:
+```
+true
+```
+
+### Nu jij:
+``` csharp runner
+using System;
+
+Console.WriteLine();
+``` 
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+>
+>bool IsWaar != true;
+>Console.WriteLine(IsWaar);
+> ```
+
```

Comment:
Opmerking: Weet niet of het een gedeeltelijke uitwerking betreft hier, maar... Wil je niet eigenlijk eerder iets zeggen van 'Console.WriteLine(!IsWaar);' ? Bovendien maakt het dan niet uit of je er true of false instopt.

Reply:
je hebt helemaal gelijk, ik scherp de uitwerking en de specificatie dat dit beter overeenkomt.